maths/max_sectors_of_circle.py

```diff
@@ -0,0 +1,28 @@
+def max_sectors_of_circle(num_cuts: float) -> float:
+    """
+    returns the maximum amount of
+    sectors a circle can be divided
+    by if cut 'num_cuts' times
+
+    >>> max_sectors_of_circle(54)
+    1486.0
+    >>> max_sectors_of_circle(7)
+    29.0
+    >>> max_sectors_of_circle(22.5)
+    265.375
+    >>> max_sectors_of_circle(-222)
+    -1
+    >>> max_sectors_of_circle("-222")
+    Traceback (most recent call last):
+    TypeError: num_cuts must be a numeric value.
+    """
+
+    if not isinstance(num_cuts, (int, float)):
+        raise TypeError("num_cuts must be a numeric value.")
+    return ((num_cuts + 2 + num_cuts**2) * (1 / 2)) if num_cuts >= 0 else -1
```

Comment:
Would this produce identical results?  It should be faster than doing the calculation each time.
```suggestion
    return ((num_cuts + 2 + num_cuts**2) * .5) if num_cuts >= 0 else -1
```

Reply:
Most definitely! Thank you very much for the review. Committed the suggestion just now :))